src/bin/main.rs

```diff
@@ -0,0 +1,83 @@
+use std::collections::BinaryHeap;
+
+use clap::Parser;
```

Comment:
Where is this being used?

Reply:
`Config::parse()`, I use _clap_'s derive macro so that the trait `Parser` is implemented and I need to bring it into scope to use it in line 15